src/code/z_parameter.c

```diff
@@ -2763,7 +2763,9 @@ void Interface_DrawItemButtons(PlayState* play) {
                         (R_ITEM_BTN_X(3) + R_ITEM_BTN_WIDTH(3)) << 2, (R_ITEM_BTN_Y(3) + R_ITEM_BTN_WIDTH(3)) << 2,
                         G_TX_RENDERTILE, 0, 0, R_ITEM_BTN_DD(3) << 1, R_ITEM_BTN_DD(3) << 1);
 
-    if ((pauseCtx->state < 8) || (pauseCtx->state >= 18)) {
+    if ((pauseCtx->state < PAUSECTX_STATE_8) /* PAUSECTX_STATE_0, PAUSECTX_STATE_1, PAUSECTX_STATE_2, PAUSECTX_STATE_3,
+                                 PAUSECTX_STATE_4, PAUSECTX_STATE_5, PAUSECTX_STATE_6, PAUSECTX_STATE_7 */
+        || (pauseCtx->state >= PAUSECTX_STATE_18_FLIP_PAGES_AND_UNPAUSE) /* PAUSECTX_STATE_18_FLIP_PAGES_AND_UNPAUSE, PAUSECTX_STATE_19_UNPAUSE */) {
```

Comment:
```suggestion
if (!((pauseCtx->state >= PAUSE_STATE_8) && (pauseCtx->state <= PAUSECTX_STATE_17))) {
```
Pause state 8 is the first "gameover" pause state, and pause state 17 is the last "gameover" pause state. This check is all over kaleido and is just checking if the state is NOT in the gameover range